README.md

```diff
@@ -1,3 +1,7 @@
+### Get ready for QHack 2024!
+
+QHack 2024 brings even more coding challenges for you to showcase your Quantum Programming skills. Solve some [PennyLane Challenges](https://pennylane.ai/challenges/) to get the gears turning. See you at QHack!
```

Comment:
Nice! Could you include the link to the QHack site?
https://qhack.ai

Reply:
Done!